tests/unit/BackendClient-spec.js

```diff
@@ -0,0 +1,183 @@
+            {
+                console.log('Promise failed');
+                done(err);
+            });
+    });
+    it ("Promise should be failed, after error occurred, in the create new session",function(done){
+        KalturaClientMock = {
+            KalturaClient : sinon.stub().returns({
+                session : {
+                    start : sinon.stub().callsArgWith(0,null,'1')
+                },
+                setSessionId: sinon.stub()
+            })
+        };
+        var mocks = {
+            './logger/logger': sinon.stub().returns(loggerMock),
+            './kaltura-client-lib/KalturaClient' : KalturaClientMock
+        };
+        var sessionDurationTest=config.get('backendClient').ksSessionRefreshIntervalMinutes* 60 * 1000;
+        var FakeTime=sessionDurationTest+1;
+        clock = sinon.useFakeTimers(FakeTime);
+        var BackendClient=createBackendClient(mocks);
+        BackendClient.SetLastSessionTime(0);
+        var promise = BackendClient.getLiveEntriesForMediaServer();
+
+        promise.then(function(){
+            done(new Error("Expected getLiveEntries to fail"));
+        }, function(){
+            done();
+        });
+    });
+    it("Calling the function .getLiveEntriesForMediaServer() twice, check that function create new session is called once!",function(done){
+        setSessionIdMock=sinon.spy();
+        KalturaClientMock = {
+            KalturaClient : sinon.stub().returns({
+                session : {
+                    start : sinon.stub().callsArgWith(0,'1')
+                },
+                setSessionId: setSessionIdMock,
+                liveStream: {
+                    listAction: sinon.stub().callsArgWith(0, '1')
+                }
+            })
+        };
+        var mocks = {
+            './logger/logger': sinon.stub().returns(loggerMock),
+            './kaltura-client-lib/KalturaClient' : KalturaClientMock
+        };
+        var FakeTime=1;
+        clock = sinon.useFakeTimers(FakeTime);
+        var BackendClient=createBackendClient(mocks);
+        expect(setSessionIdMock.callCount).to.eql(0);
+        BackendClient.getLiveEntriesForMediaServer()
+            .then(function(){
+                expect(setSessionIdMock.callCount).to.eql(1);
+                clock = sinon.useFakeTimers(FakeTime+1);
+                return BackendClient.getLiveEntriesForMediaServer();
+            })
+            .then(function(){
+                expect(setSessionIdMock.callCount).to.eql(1);
+                done();
+            }).catch(function(err){
+                done(err);
+            })
+    });
+    it("Check that getLiveEntriesForMediaServer's return object has an appropriate form",function(done){
+        item_obj_example1={
+            dvrStatus:0,
+            id:"0_aaa"
+        };
+        item_obj_example2={
+            dvrStatus:1,
+            id:"0_bbb"
+        };
+        objsTest=[];
+        objsTest.push({"dvrWindow":100,"entryId":"0_aaa"});
```

Comment:
you can just create it verbatim - [{"dvrWindow":100,"entryId":"0_aaa"}, {"dvrWindow":7200,"entryId":"0_bbb"}]